plugins/editor/class.editor.plugin.php

```diff
@@ -1466,7 +1488,10 @@ public function settingsController_editor_create() {
     public function setup() {
         \Gdn::config()->touch([
             'Garden.MobileInputFormatter' => 'TextEx',
-            'Plugins.editor.ForceWysiwyg' => false
+            'Plugins.editor.ForceWysiwyg' => false,
+            'ImageUpload.Limits.Enabled' => false,
```

Comment:
These new configuration values should go into `config-defaults.php` since they don't particularly belong to the Advanced Editor.

For future reference we should also avoid writing configuration values on setup in the future. Having a default value is almost always preferable.